libs/community/langchain_community/storage/astradb.py

```diff
@@ -126,7 +126,7 @@ async def ayield_keys(self, *, prefix: Optional[str] = None) -> AsyncIterator[st
 
 
 @deprecated(
-    since="0.1.24",
+    since="0.0.22",
```

Comment:
@efriis Other components such as [Pinecone VectorStore](https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/vectorstores/pinecone.py#L46) or [AstraDB VectorStore](https://github.com/langchain-ai/langchain/blob/master/libs/community/langchain_community/vectorstores/astradb.py#L69) have been deprecated with the version of langchain-core when moved from to the partner package.
Shouldn't the langchain-community version be used for them too ?
I can make the PR.